### PR TITLE
make it compile under aarch64

### PR DESCRIPTION
**Pull request by chincheta0815**
I hopefully found your latest repo...
These changes should make is easily possible to compile on aarch64 (RPI 64bit) which needs to use luajit-2.1.
It would be awesome if you could respect those changes.

### CODE REVIEW

File: lib-src/luaexpat-1.2.0/src/lxplib.c [hunk 1]
```diff
@@ -517,7 +529,11 @@ static int lxp_stop (lua_State *L) {
   return 1;
 }
 
+#if defined(LUAJIT_VERSION) && LUAJIT_VERSION > 20
+static const struct luaL_Reg lxp_meths[] = {
```
Comment (blaisedias) [inline lib-src/luaexpat-1.2.0/src/lxplib.c:533]: Hey @chincheta0815, why do we need changes of this type where all we are doing is changing the name of the struct but not the contents/shape of the struct ?
IMHO the change would be smaller, and simpler with fewer #ifdefs (which is always a good thing)

Comment (chincheta0815) [reply lib-src/luaexpat-1.2.0/src/lxplib.c:533]: I totally agree with you, but I do not know how to account for a simple change in a lower/upper letter case change in a different way...

Comment (blaisedias) [reply lib-src/luaexpat-1.2.0/src/lxplib.c:533]: ah okay,

**Comment by blaisedias** [comment]
@chincheta0815  Thanks. I have merged your PR

**Pull request merged**